tests/app/forms/handlers/test_duration_field.py

```diff
@@ -0,0 +1,29 @@
+from wtforms import FormField
+
+from app.forms.field_handlers.duration_handler import DurationHandler
+
+
+def test_get_field():
+    date_json = {
+        'guidance': '',
+        'id': 'year-month-answer',
+        'label': 'Duration',
+        'mandatory': True,
+        'options': [],
+        'q_code': '11',
+        'type': 'Duration',
+        'units': ['years', 'months'],
+        'validation': {
+            'messages': {
+                'INVALID_DURATION': 'The duration entered is not valid.  Please correct your answer.',
+                'MANDATORY_DURATION': 'Please provide a duration to continue.',
+            }
```

Comment:
Can `validation` be dropped if it's not used in the test?

Reply:
Thinking about this...doesn't it imply that a duration field can't be marked as mandatory?

Reply:
The validators for the duration field are created within the form rather than passed to it.

Reply:
To answer my own question, no - because duration_form uses a custom .validate() that means validation is done at the form and not the field level.